usage-collection/src/main/proto/usage.proto

```diff
@@ -1171,3 +1175,77 @@ message CDPUDXBundleDetails {
   // The type of the diagnostics bundle.
   string type = 5;
 }
+
+//Autoscale Policy Type
+message AutoscalePolicyType {
+  enum Value {
+    // A value indicating the enum is unset.
+    UNSET = 0;
+    // The schedule-based autoscale policy.
+    TIME_BASED = 1;
+    // The load-based autoscale policy.
+    LOAD_BASED = 2;
+  }
+}
+
+//Autoscale Policy Definition
+message CDPAutoscalePolicyDefinition {
+  // The autoscale policy type
+  AutoscalePolicyType.Value autoscalePolicyType = 1;
+  // The autoscale policy parameters
+  string autoscalePolicyParameters = 2;
+  // The autoscale host group
+  string autoscalingHostGroup = 3;
+  // The autoscale policy name.
+  string autoscalingPolicyName = 4;
+}
+
+//Autoscale Trigger Status
+message AutoscaleScalingStatusType {
+  enum Value {
+    // A value indicating the enum is unset.
+    UNSET = 0;
+    // Scaling action successfully requested.
+    SUCCESS = 1;
+    // Scaling action failed.
```

Comment:
What's the difference between FAILED and TRIGGER_FAILED?

Reply:
Updated the status for easier understanding.